src/apollo/fragments.ts

```diff
@@ -1,4 +1,12 @@
-// import gql from "@apollo/client";
+import { gql } from "@apollo/client";
+
+export const accountErrorFragment = gql`
+  fragment AccountError on AccountError {
```

Comment:
```suggestion
  fragment AccountErrorFragment on AccountError {
```

I find adding fragment suffix improves reading queries. @jwm0 whats your opinion?

Reply:
Also, having a variable named exactly as fragment helps during debugging

Reply:
Exactly, adding `Fragment` part is a good point. I've completely copied the mutations, queries and fragments from the master so feel free to propose changes.